### DISCUSSION TITLE
Change/Add Localisation

### DISCUSSION THREAD

**Discussion by FischiTheDev** [ppy/osu#16303] [Q&A] [answered]
Hi. I am wondering if you could change/add localisations to osu!lazer or remove them. If yes then how because I can't find them in osu.Game/Localisation.
Regards, Fischi :)

**Comment by frenzibyte** [comment, marked as answer]
You can contribute to the localisations via [the crowdin project](https://crowdin.com/project/osu-web). (`client` is dedicated for osu!(lazer), while `master` is used by both osu!(lazer) and osu! web)

**Reply by FischiTheDev** [reply]
Can I change the crowdin project? I want to add some custom languages that normally wouldnt be added into the game.

**Reply by bdach** [reply]
what does "custom languages" mean?

**Reply by FischiTheDev** [reply]
Like languages that do exist but aren't available in the Crowdin project.

**Reply by peppy** [reply]
If there's a language that's not already there, it likely means the demand is low for it. You can ask for adding a language here though, and we will assess if it's something we want to do.
